src/libpriv/rpmostree-passwd-util.c

```diff
@@ -850,44 +767,35 @@ _rpmostree_gfile2stdio (GFile         *source,
                         GCancellable  *cancellable,
                         GError       **error)
 {
-  gboolean ret = FALSE;
   gsize len;
-  FILE *src_stream = NULL;
+
+  *ret_src_stream = NULL;
```

Comment:
Mmmm...I try to ensure output vars are always initialized on successful returns.  Part of a general philosophy going back a long time.  It's true that using autocleanups basically forces one to initialize anyways though.